addons/mail/static/src/components/message_reaction_group_summary_view/message_reaction_group_summary_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageReactionGroupSummaryView" owl="1">
+        <div class="o_MessageReactionGroupSummaryView mr-1 mb-1"
+             t-att-class="{ 'o-isHighlighted': messageReactionGroupSummaryView and messageReactionGroupSummaryView.messageReactionGroup === messageReactionGroupSummaryView.messageReactionsSummaryView.highlightedReaction }"
+             t-on-click="messageReactionGroupSummaryView and messageReactionGroupSummaryView.onClickReaction(messageReactionGroupSummaryView.messageReactionGroup)"
+             t-attf-class="{{ className }}"
+             t-ref="root">
+            <t t-if="messageReactionGroupSummaryView">
+                <span class="o_MessageReactionGroupSummaryView_reactionEmote mx-1" t-esc="messageReactionGroupSummaryView.messageReactionGroup.content"/>
+                <span class="o_MessageReactionGroupSummaryView_reactionCount  mx-1" t-att-class="{ 'o-isHighlighted': messageReactionGroupSummaryView.messageReactionGroup === messageReactionGroupSummaryView.messageReactionsSummaryView.highlightedReaction }" t-esc="messageReactionGroupSummaryView.messageReactionGroup.count"/>
```

Comment:
`messageReactionGroup` is a poor name as it's basically a reaction to a message. The `group` suffix has been added because of the number next to the reaction to specify how many people reacted the same. IMO `messageReactionGroup` should be renamed `messageReaction`and maybe we should add a component to manage the counting

Reply:
`messageReactionGroup` is a good name because it holds the aggregated information (hence the "group" idea). A `messageReaction` is one occurrence by one user, and its name is correct too.

Here it's `highlightedReaction` that is wrongly named, because it contains a group instead.

Reply:
I would have split the ideas of reaction and group in two separated models. It does work good here but I still feel strange to not have a proper model `messageReaction`.
Do you want me to rename `highlightedReaction` in `highlightedReactionGroup` ?